.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+# This is a basic workflow to help you get started with Actions
+
+name: XGoost-CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on: [push, pull_request]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  test-with-R:
+    runs-on: ${{ matrix.config.os }}
+
+    name: Test R on OS ${{ matrix.config.os }}, R (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: windows-latest, r: 'release'}
+          # how to install omp on osx?
```

Comment:
The easiest way is to use Homebrew: `brew install libomp`.

Reply:
We should keep using Travis CI for testing Mac, because GitHub Actions doesn't support MacOS High Sierra, whereas Travis CI does.

Reply:
Got it, I will remove the comment.